testing/tests/ui/comments-in-macro-calls.rs

```diff
@@ -0,0 +1,29 @@
+use askama::Template;
+
+// Regression test for <https://issues.oss-fuzz.com/issues/425145246>.
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(// hello) }}")]
+struct LineComment;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(/* hello */) }}")]
+struct BlockComment;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(/// hello) }}")]
+struct OuterLineDoc;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(/** hello) }}")]
+struct OuterBlockDoc;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(//! hello) }}")]
+struct InnerLineDoc;
+
+#[derive(Template)]
+#[template(ext = "html", source = "{{ e!(/*! hello) }}")]
```

Comment:
What happens for this case:

```
{{ e!(//)
}}
```

?

Reply:
> error: expected `)` but found `}`

The line comment `//` make the parser skip the closing paren `)`, so the next character it sees is `}`, but it expects `)` to close the macro call.

I added tests to ensure that behavior (and found a typo in my implementation this way, oops).